tests/leagueResource.spec.js

```diff
@@ -122,28 +122,28 @@ describe("resource: leagueResource", function() {
     );
   });
 
-  // players
-  it("should build a proper url to retrieve players via a league key", function() {
-    nock("https://fantasysports.yahooapis.com")
-      .get("/fantasy/v2/league/328.l.34014/players?format=json")
-      .reply(200, {});
+  // // players
```

Comment:
This was [commented out in the source](https://github.com/whatadewitt/yfsapi/blob/master/resources/leagueResource.mjs#L178), so there is nothing to test